README.md

```diff
@@ -25,11 +26,21 @@ which allow for:
 When the function has resolved the library will return either the manipulated context object to make assertions
 against or the returned value *if* the special `$return` name is used for one of the output bindings.
 
-If you follow the "ordered argument" pattern then you'll need to define the triggers/outputs in the same order
+If you follow the "ordered argument" pattern then you'll need to define the triggers in the same order
 as in your `function.json`.
 
-By default, logging uses `console` as a backend, although you can import and use `setContextLogger` to set your own.
-Your logger will need to conform to the same interface as `console` and will be wrapped to work with the `context.log` interface.
+By default, logging uses `console`, you can change this by manipulating the context object before it is passed to
+your function:
```

Comment:
I think that console logger should be disabled by default.
It might be useful for debugging, but nobody wants to see debug output when running full test suite.

Reply:
I think saying "nobody" wants to see console output in test suites is a bit much. If your test suite stubs out your logger so it doesn't produce any output during tests, then I don't think it's too much to stub it out in the function runner either.

For me, I'd say the console output during test suites boarders on essential.

Perhaps it just needs to be easier to stub it out?

Reply:
Yes, a bit easier would be nice.

Now I'm trying to find a good place for NullLogger in my code base, maybe you could serve it as a part of this library?
```ts
const setNullLogger = (context: Context) => {
  context.log = Object.assign(() => {}, {
    info() {},
    error() {},
    verbose() {},
    warn() {},
  });
};
```